api/utils/requestProcessor.js

```diff
@@ -2068,7 +2068,7 @@ const processRequest = (params) => {
                     }
                     else {
                         if (params.qstring.event && params.qstring.event.startsWith('[CLY]_group_')) {
-                            validateRead(params, 'core', countlyApi.data.fetch.fetchMergedEventGroups, params.qstring.method);
```

Comment:
did you remove **params.qstring.method** intentionally?
if so why?

Reply:
Yes this was done intentionally. This is a bug which is there only on feature/new-ui. According to the internal implementation this param is not to be passed..else the api aint working. This can be checked currently on ui.count.ly too.

Reply:
okay.